src/chronos/chronos_bolt.py

```diff
@@ -429,6 +429,7 @@ class ChronosBoltPipeline(BaseChronosPipeline):
     default_context_length: int = 2048
 
     def __init__(self, model: ChronosBoltModelForForecasting):
+        assert isinstance(model, PreTrainedModel)
```

Comment:
Actually let me do this otherwise, I don't like it. This must be a type-checker issue, not ours